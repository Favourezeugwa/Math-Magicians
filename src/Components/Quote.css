.quote-container {
  border-radius: 20px;
  position: absolute;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
  width: 600px;
  padding: 50px;
  text-align: center;
  background-color: #1a21297a;
}

span {
  color: #ff5c33;
}
