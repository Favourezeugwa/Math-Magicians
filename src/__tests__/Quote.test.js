import renderer from 'react-test-renderer';
import Quote from '../Components/Quote';

it('Test quote component renders correctly', () => {
  const tree = renderer
    .create(<Quote />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
