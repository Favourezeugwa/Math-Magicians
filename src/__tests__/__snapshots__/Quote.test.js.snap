// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test quote component renders correctly 1`] = `
<div
  className="quote-container"
>
  <p
    className="quote-header"
  >
    Lorem ipsum dolor sit amet consectetur, adipisicing elit. Aut earum suscipit harum, ipsa, autem numquam laboriosam iusto labore necessitatibus illum, ducimus impedit quae voluptate corporis quod. Qui repudiandae, quod commodi, praesentium quidem esse accusamus aliquam tota porro asperiores nostrum sapiente sunt tenetur rem itaque nesciunt provident soluta minvero.
    <br />
    <span>
      <i>
        - William Paul Thompson
      </i>
    </span>
  </p>
</div>
`;
